pkgs/development/compilers/souffle/default.nix

```diff
@@ -19,6 +19,11 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-wdTBSmyA2I+gaSV577NNKA2oY2fdVTGmvV7h15NY1tU=";
   };
 
+  patches = lib.optionals stdenv.isDarwin [ ./threads.patch ]
+    ++ [ ./missing-override.patch ];
```

Comment:
Could we fetchpatch them instead?

Reply:
Sure. I squashed and rebased on master already.